packages/http-client-csharp/emitter/src/sdk-context-options.ts

```diff
@@ -1,7 +1,9 @@
 import { CreateSdkContextOptions } from "@azure-tools/typespec-client-generator-core";
 
 export let defaultSDKContextOptions: CreateSdkContextOptions = {
-  versioning: undefined,
+  versioning: {
```

Comment:
By default, TCGC will following Azure preview/stable apiversion filter, MGC is common emitter, it should not follow this. So set the Regex which will not match any apiVersion to filter.

Reply:
Then what do we do in `autorest.csharp`? Shall we restore preview/stable api version filter?

Reply:
We will set the filter back in autorest.csharp emitter. See the autorest.csharp regen preview pr